packages/client/src/sdk.js

```diff
@@ -74,7 +73,6 @@ export default {
   fetchData,
   QueryUtils,
   ContextScopes: Constants.ContextScopes,
-  getAPIKey,
```

Comment:
This is unused in our code, but the SDK is provided to plugin authors. This was added to allow them to authenticate with the public API more easily, so it needs to stay.

Reply:
Great, I added a comment to avoid future removals